packages/hardhat/contracts/VNDToken.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract VNDToken is ERC20 {
+	// 1. Token Basics
+	uint256 public constant MAX_SUPPLY = 50_000_000 * 10 ** 18; // 100 million tokens
```

Comment:
The comment here doesn't match the amount. I assume it'll be `// 50 million tokens`